ztp/ran-site-plan-crd/site-config-crd.yaml

```diff
@@ -0,0 +1,298 @@
+                          type: object
+                          required:
+                            - cidr
+                          properties:
+                            cidr:
+                              description: CIDR is the IP block address pool.
+                              type: string
+                            hostPrefix:
+                              description: |
+                                HostPrefix is the prefix size to allocate to each
+                                node from the CIDR. For example, 24 would allocate
+                                2^8=256 adresses to each node. If this field is
+                                not used by the plugin, it can be left unset.
+                              type: integer
+                              format: int32
+                      machineNetwork:
+                        description: MachineNetwork is the list of IP address pools for machines.
+                        type: array
+                        items:
+                          description: MachineNetworkEntry is a single IP address block for node IP blocks.
+                          type: object
+                          required:
+                            - cidr
+                          properties:
+                            cidr:
+                              description: CIDR is the IP block address pool for machines within the cluster.
+                              type: string
+                      serviceNetwork:
+                        description: |
+                          ServiceNetwork is the list of IP address pools for services default is 172.30.0.0/16.
+                          currently only one entry is supported.
+                        type: array
+                        maxItems: 1
+                        items:
+                          type: string
+                      apiVIP:
+                        description: APIVIP is the virtual IP used to reach the OpenShift cluster's API.
+                        type: string
+                      ingressVIP:
+                        description: IngressVIP is the virtual IP used for cluster ingress traffic.
+                        type: string
+                      manifestsConfig:
+                        description: day0 required manifestsConfig.
+                        type: object
+                        properties:
+                          ntpServer:
+                            description: ntpServer required by chrony manifest.
+                            type: string
+                          cpuset:
+                            description: cpuset required by workload partitioning manifest.
+                            type: string
+                          managementWorkloadName:
+                            description: managementWorkloadName required by workload partitioning manifest.
+                            type: string
```

Comment:
The only valid value for this is "management" right now.  I don't think we know enough about what other workloads might exist in the future to bother making this configurable yet.

But do we want a boolean to enable/disable management workload partitioning entirely? (@browsell ?)

Reply:
okay, we can remove it. Just want to avoid the need for it later

Reply:
Yeah, I think it's too soon to know enough about how this will be configurable in the future; let's delete it for now.

Reply:
+1, delete for now